cosmo_tester/framework/test_hosts.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, image_type, test_config, bootstrappable=False):
         self.api_ca_path = None
         self.enable_ssh_wait = True
         self.should_finalize = True
+        self.should_teardown = True
```

Comment:
This won't work, because this means that we'll tear down just the db service and then mark the node as not needing to tear down. The 'should_teardown' flag is essentially the installed_configs list, so something needs to be done with that.